src/main/java/com/airbnb/kafka/KafkaStatsdMetricsReporter.java

```diff
@@ -23,119 +21,130 @@
 import com.airbnb.metrics.ExcludeMetricPredicate;
 import com.airbnb.metrics.MetricDimensionOptions;
 import com.airbnb.metrics.StatsDReporter;
+import com.timgroup.statsd.NonBlockingStatsDClient;
+import com.timgroup.statsd.StatsDClient;
+import com.timgroup.statsd.StatsDClientException;
 import com.yammer.metrics.Metrics;
 import com.yammer.metrics.core.MetricPredicate;
 import com.yammer.metrics.reporting.AbstractPollingReporter;
 import kafka.metrics.KafkaMetricsReporter;
 import kafka.utils.VerifiableProperties;
-import org.apache.log4j.Logger;
+import org.slf4j.LoggerFactory;
 
 import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicBoolean;
 
 /**
- * Kafka uses Yammer Metrics for metrics reporting in both the server and the client.
- * This can be configured to report stats using pluggable stats reporters to hook up to your monitoring system.
- *
- * This class allows to report Kafka Metrics to StatsD.
- *
- * The reporter is enabled and configured from the Kafka config file, server.properties.
  *
- * @see kafka.metrics.KafkaMetricsReporter
  */
 public class KafkaStatsdMetricsReporter implements KafkaStatsdMetricsReporterMBean, KafkaMetricsReporter {
 
+    private static final org.slf4j.Logger log = LoggerFactory.getLogger(StatsDReporter.class);
 
-    public static final String DEFAULT_EXCLUDE_REGEX = "(kafka\\.consumer\\.FetchRequestAndResponseMetrics.*)|(.*ReplicaFetcherThread.*)|(kafka\\.server\\.FetcherLagMetrics\\..*)|(kafka\\.log\\.Log\\..*)|(kafka\\.cluster\\.Partition\\..*)";
```

Comment:
would you exclude or include the matches? having `EXCLUDE`  in the name is unambiguous.